internal/exec/util/file.go

```diff
@@ -191,6 +201,46 @@ func (u Util) PerformFetch(f *FetchOp) error {
 	return nil
 }
 
+func (u Util) GetUserGroupID(l *log.Logger, user types.NodeUser, group types.NodeGroup) (*int, *int) {
```

Comment:
Now that this function exists, it's duplicating a lot of logic in [`PrepareFetch`](https://github.com/coreos/ignition/pull/422/files#diff-a20394991caacd494d8a57e433810c95R88), can you make `PrepareFetch` use this function too?